client/src/components/RouterController/RouterController.js

```diff
@@ -45,6 +45,7 @@ function RouterController() {
     {
       exact: true,
       path: '/cart',
```

Comment:
Better to create a constant for all routes.

Reply:
RouterController sounds like a service that will be handling routing, if you just have a list of routers, a better name would be AppRoutes.